public/index.html

```diff
@@ -42,11 +45,11 @@
       // Link the logos
       const faviconLink = document.createElement('link');
       faviconLink.rel = 'icon';
-      faviconLink.href = `%PUBLIC_URL%/${subdomain}/favicon.ico`;
+      faviconLink.href = `assets/${subdomain}/favicon.ico`;
```

Comment:
Testing locally this changes breaks a few places where site data is acquired from the manifest. For example the app-install pop-up which uses the `PUBLIC_URL` environment variable. For example `const logoURL = `${process.env.PUBLIC_URL}/${subdomain}/logo192.png`;`

Please update the places where `PUBLIC_URL` if that variable is being retired.

Reply:
Yes, so after confirming with Rohit, the way that the deployment on fv-app works, is that all the files from the specific env config are copied to the `public` folder which are then served by the webpack config in this repo.
Could you please give that a try as well ? e.g. for `dev` env, copying files from `dev-config/public` folder into `public` folder here, and it should work as intended. 
I can also jump on a call to do the process together if required. Thank you :)